pkg/controllers/nodeclaim/capacityreservation/controller.go

```diff
@@ -130,6 +130,7 @@ func (c *Controller) syncCapacityType(ctx context.Context, capacityType string,
 	if err != nil {
 		return false, fmt.Errorf("listing nodes for nodeclaim %q, %w", nc.Name, err)
 	}
+	updated := false
```

Comment:
We also want to set updated to true if we successfully updated the nodeclaim on line 121, right?